bin/citrea/configs/bitcoin-regtest/prover_rollup_config.toml

```diff
@@ -20,6 +20,9 @@ path = "prover-db"
 # the host and port to bind the rpc server for
 bind_host = "127.0.0.1"
 bind_port = 12346
+max_request_body_size = 10485760 # 10 MB
```

Comment:
since we've set the default values in `config.rs` I don't think we need to put the default values here as well in the rollup_config.toml'S

Reply:
can describe those in the docs